tests/testColor.py

```diff
@@ -308,11 +275,6 @@ def defineFilterProperty(self, name, lambdaEff, force=False):
     def testListFilters(self):
         self.assertEqual(afwImage.Filter_getNames(), self.filters)
 
-    def testCtor(self):
```

Comment:
As I say above, `asserts` are not required in test code. It may be that the test code used to throw an exception but then something was fixed. Of course, if everything that is in this test is found in other tests then it is no longer needed.